src/web/pages.rs

```diff
@@ -36,17 +36,17 @@ lazy_static! {
                 Ok(c) => c,
                 Err(e) => {
                     println!("Template parsing error(s): {e}");
-                    ::std::process::exit(1);
+                    panic!();
                 }
             };
             match tera.add_raw_template(&path, contents) {
                 Ok(t) => t,
                 Err(e) => {
                     println!("Template parsing error(s): {e}");
-                    ::std::process::exit(1);
+                    panic!();
```

Comment:
`panic!` prints out whatever's in the parenthesis, so this should be:
```rust
panic!("Template parsing error(s): {e}");
```

Reply:
ah! fixed